feature/course/src/main/java/com/teamsolply/solply/course/CourseViewModel.kt

```
@@ -0,0 +1,11 @@
+package com.teamsolply.solply.course
+
+import com.teamsolply.solply.ui.base.BaseViewModel
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+
+@HiltViewModel
+class CourseViewModel @Inject constructor() : BaseViewModel<CourseState, CourseIntent, CourseSideEffect>(CourseState()) {
+    override fun handleIntent(intent: CourseIntent) {
+    }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)